css/style.css

```diff
@@ -0,0 +1,1255 @@
+  width: 100%;
+  height: 200px;
+}
+
+.ditem {
+  margin-bottom: 20px;
+}
+
+.dealtitlediv {
+  justify-content: space-between;
+  align-items: center;
+}
+
+.imgcont {
+  overflow: hidden;
+}
+
+.ptag {
+  position: absolute;
+  top: -20px;
+  right: -110px;
+  transform: rotate(45deg);
+  background-color: #ffe712;
+  height: 25px;
+  width: 250px;
+  padding-top: 50px;
+  text-align: center;
+}
+
+/* styling for footer section */
+
+footer {
+  background-color: #1a1a1a;
+  padding: 2% 0 0 0;
+  text-align: center;
+}
+
+.iconcont {
+  text-align: center;
+  margin: 0 auto;
+}
+
+.ficon {
+  color: white;
+  margin: 8px;
+  font-size: 100%;
+  background-color: #333;
+  padding: 8px;
+  border-radius: 50%;
+  border: 2px solid #333;
+}
+
+.ficon:hover {
+  background-color: black;
+  border: 2px solid grey;
+}
+
+.footnav {
+  text-align: center;
+  margin: 2% auto;
+  font-size: 75%;
+  padding-bottom: 20px;
+  justify-content: center;
+}
+
+.footnav li {
+  list-style-type: none;
+}
+
+.footnav a:hover {
+  color: white;
+}
+
+.footbottom {
+  background-color: #000;
+  padding: 2% 0;
+  text-align: center;
+}
+
+#tnwfooter {
+  width: 50px;
+  height: 25px;
+  margin-right: 5px;
+}
+
+/* responsiveness for medium devices */
+@media only screen and (max-width: 1023px) and (min-width: 768px) {
+  /* top navigation styling */
+  .navtop {
+    background-color: black;
+    padding: 10px;
+    font-size: 78%;
+    height: 30px;
+  }
+
+  .d-m-block {
+    display: block;
+  }
+
+  .navlist li {
+    margin-right: 5px;
+  }
+
+  /* navlogom&s style */
+  .navlogom-m {
+    background: linear-gradient(to bottom right, #c09, #f42 85%);
+    width: 106px;
+    height: 60px;
+    text-align: center;
+    padding-top: 30px;
+    position: absolute;
+    top: 0;
+    z-index: 200;
+  }
+
+  .navlist {
+    margin-left: 80px;
+  }
+
+  .navlogom-m img {
+    width: 70%;
+  }
+
+  .navlogom-s img {
+    width: 70%;
+  }
+
+  /* navigation fixed styling */
+  .header {
+    position: sticky;
+    top: 0;
+    width: 100%;
+    z-index: 100;
+  }
+
+  .navfixed {
+    height: 40px;
+    font-family: "Nova Square", cursive;
+  }
+
+  .navli2 {
+    margin-left: -60px;
+  }
+
+  #searchbtn {
+    border-bottom: 1px rgba(84, 84, 84, 0.27) solid;
+    padding-bottom: 10px;
+    width: 25px;
+    padding-left: 15px;
+    font-size: 110%;
+  }
+
+  .hiddenmd {
+    display: none;
+  }
+
+  /* yellow nav styling */
+
+  .yellow-nav {
+    background-color: #ffe712;
+    height: 20px;
+    padding: 5px 5px 5px 105px;
+    clear: both;
+  }
+
+  /* text styling for medium screen */
+  .textstyle1 {
+    color: white;
+    font-size: 170%;
+    text-align: justify;
+  }
+
+  .textstyle2 {
+    color: white;
+    font-size: 70%;
+    text-align: justify;
+  }
+
+  .textstyle3 {
+    color: white;
+    font-size: 90%;
+    text-align: justify;
+  }
+
+  /* logo sizing */
+
+  .logo-TNW img {
+    display: none;
+  }
+
+  .logo-TNW p {
+    display: none;
+  }
```

Comment:
Group selectors with the same style definition

Reply:
We have tried to group selectors with the same value. Removed ID selectors and instead used classes and we have also added the missing footer text.